.ruby-style.yml

```
@@ -550,7 +550,7 @@ Style/HashSyntax:
                  Prefer Ruby 1.9 hash syntax { a: 1, b: 2 } over 1.8 syntax
                  { :a => 1, :b => 2 }.
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#hash-literals'
-  Enabled: false
+  Enabled: true
```

```suggestion
  Enabled: true
  EnforcedStyle: ruby19_no_mixed_keys
```

This **not** the Rubocop default enforced style but matches: https://github.com/everypolitician/everypolitician-data/blob/b63842260aafecfe6c67b1268a1e517d9b18d957/.rubocop_base.yml#L42

---

👍 , though now's the time to consider https://samphippen.com/hash-rockets-are-good-actually